lib/components/Input/text-field/text-field.stories.tsx

```diff
@@ -0,0 +1,160 @@
+import { Meta, StoryObj } from "@storybook/react";
+import TextField from ".";
+import {
+    StandaloneCircleCheckBoldIcon,
+    StandaloneTriangleExclamationBoldIcon,
+} from "@deriv/quill-icons";
+
+const meta = {
+    title: "Components/Inputs/Text Field",
+    component: TextField,
+    parameters: {
+        layout: "centered",
+    },
+    tags: ["autodocs"],
+    args: {
+        inputSize: "md",
+        status: "neutral",
+        disabled: false,
+        variant: "fill",
+        textAlignment: "left",
+        label: "",
```

Comment:
Do we need empty strings? It's falsy values, will be considered as undefined anyway